digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala

```diff
@@ -2157,4 +2164,205 @@ class ProjectServiceSpec  extends FunSuite with Matchers with BeforeAndAfter {
       updatedLinks.maxBy(_.endAddrMValue).id should be (updatedLinks.find(_.discontinuity.value == 2).get.id)
     }
   }
+
+  test("Calculate split road addresses with Unchanged + Term/New") {
+    val projectDate = Some(DateTime.parse("2021-01-01"))
+    val roadAddress = RoadAddress(123L, 5L, 8L, RoadType.PublicRoad, Combined, Continuous, 1080L, 1137L,
```

Comment:
Please be careful when using fixed numbers for ID's, try to use the best sequence for them:
VIITE_GENERAL_SEQ;
VIITE_PROJECT_SEQ;
LRM_POSITION_PRIMARY_KEY_SEQ;
PRIMARY_KEY_SEQ;